### ISSUE TITLE
Navigator does not update to removed series

### ISSUE THREAD

**Issue by polarathene** [highcharts/highcharts#3767]
I've noticed that when a series is added via `chart.addSeries()` Navigator responds changing it's length if needed. The supplied jsFiddle does not provide much of a visual scale difference other than the line being more simplified(decreased points via _dataGrouping_ or change to _spline_?). 

http://jsfiddle.net/465qL5f2/
Press the `Add series` button then scroll the navigator to the far left, then resize the right handle towards the left, you'll notice it jump from _September 2003_ to _1995_ which will scale quickly to _1970_. With the console open you'll also notice before/after values. Press `Remove series` button and notice how the navigators first point is still at an `x` value of **1**. The navigator extended it's range from additional series but did not trim itself when that series was removed.

This error is most likely to occur on datetime type graphs when a given `x` value is undefined/null defaulting to 0/1970.

**Comment by polarathene** [comment]
I've changed the dataset to just a 2D array with less than 300 points(cropThreshold doesn't appear to do anything though?), I have `dataGrouping` *disabled* but `forced` set to *true*. This produces a more obvious visual update to the navigator, as well as the same error described above. 

However if I change `forced` to *false*, the only visual difference is a *Jan '14* tag that is slightly out of the navigator bounds. When removing the series it returns to normal. Besides the issue with the offset label(that probably shouldn't appear), this seems like the desirable behaviour.
http://jsfiddle.net/465qL5f2/2/

I tried to apply that solution to the original fiddle example but had no luck. Something else is affecting it, it may be the way it's getting the data or the format of the data?
http://jsfiddle.net/465qL5f2/3/
